### DISCUSSION TITLE
Use Key-Up instead of Key-Down

### DISCUSSION THREAD

**Discussion by skjerns** [Flow-Launcher/Flow.Launcher#513] [Ideas]
I copy this from https://github.com/Wox-launcher/Wox/issues/2722, with hope that Flow can implement it.

I have the Windows key as a hotkey for Wox, and disabled it to display start-menu via [ClassicShell](http://www.classicshell.net/).

However, it always pops up on key-down when I do any other Windows-hotkey. For most hotkeys (win+r, win+d, ...) this is no problem, however for others (win+left/right) this does not work very well, and they become a bit clumsy to use (ie double win+right).

 I would like to know if it would be easy to implement showing Flow on key-up instead of key-down?

**Comment by jjw24** [comment]
Related:
https://github.com/jjw24/Wox/issues/138:
![image](https://user-images.githubusercontent.com/26427004/89144660-ad23cc00-d591-11ea-86d9-f60018249010.png)

From NHotkey lib author:
![image](https://user-images.githubusercontent.com/26427004/89144696-c88ed700-d591-11ea-934a-2e302d5db90e.png)
https://github.com/thomaslevesque/NHotkey/issues/17

**Comment by jjw24** [comment]
@skjerns hey how's it going. This doesnt look like a quick change we can implement. We will need to look at different lib for capturing hotkeys, so I wont be addressing this issue in the coming months or so. Hopefully another contributor would like to pick this up and have a go.

I would like to keep this open and come back to work on it later on as the suggestion is a good one, especially if we can use the Win key.
